### PR TITLE
[Monthly release] v0.4.10

### PR DESCRIPTION
**Pull request by ddoyle2017**
📆 Monthly release for all the images!

Changelog:
- [[anaconda] - setuptools, imagecodecs - apply security patch](https://github.com/devcontainers/images/pull/1217)
- [Fix the use of deprecated set-output command](https://github.com/devcontainers/images/pull/1223)
- [[python] - add 3.13 - manifest, dockerfile, readme](https://github.com/devcontainers/images/pull/1221)
- [[anaconda] - remove patches when similar or higher versions are available from upstream](https://github.com/devcontainers/images/pull/1228)
- [[Miniconda] - remove stale patches (when higher version is available from upstream)](https://github.com/devcontainers/images/pull/1229)
- [Automated update for image history](https://github.com/devcontainers/images/pull/1235)
- [[anaconda] - Revert packages version imagecodecs and werkzeug](https://github.com/devcontainers/images/pull/1254)

This PR increments the patch version for all the images and adds changes that weren't properly included in `v0.4.9`

### CODE REVIEW

File: src/anaconda/manifest.json
Comment (eljog) [inline src/anaconda/manifest.json:1]: Why is this PR do not include any `devcontainer-lock.json` for any of the images? Are they all upto date? Can you please make sure?

**Review by eljog — APPROVED** [review]

File: src/anaconda/manifest.json
Comment (ddoyle2017) [reply src/anaconda/manifest.json:1]: I re-ran the upgrade script for each of the images, which didn't produce any `devcontainer-lock.json` changes. So tentatively, this looks correct

Comment (ddoyle2017) [reply src/anaconda/manifest.json:1]: I ended up deleting all of the lock files and regenerating them, which didn't produce any diffs so I'm fairly certain they're all up to date 👍

**Comment by ddoyle2017** [comment]
Holding off on merging while we investigate an issue with the `build and push "dev" images` action

**Review by eljog — APPROVED** [review]
⚡

**Review by preechapon-cmd — APPROVED** [review]

**Pull request merged**